src/main/java/mate/academy/shop/controllers/order/OrderDetailController.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.shop.controllers.order;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.shop.lib.Injector;
+import mate.academy.shop.model.Order;
+import mate.academy.shop.model.Product;
+import mate.academy.shop.service.OrderService;
+
+@WebServlet("/order/info")
+public class OrderDetailController extends HttpServlet {
+    private static final Injector INJECTOR = Injector.getInstance("mate.academy");
+    private OrderService orderService = (OrderService) INJECTOR.getInstance(OrderService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long orderId = Long.parseLong(req.getParameter("id"));
+        Order order = orderService.get(orderId);
+        List<Product> products = order.getProductsInOrder();
```

Comment:
getProductsInOrder() just a simple getter for List<Product>? then let's name it just getProducts()